pom.xml

```diff
@@ -19,7 +19,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-shade-plugin</artifactId>
-				<version>3.3.0</version>
```

Comment:
Is there a reason, why you backported the maven-shade-plugin?

Reply:
I can't build it with the 3.3.0 version